lib/actions/local/configure.js

```diff
@@ -0,0 +1,285 @@
+/*
+Copyright 2017 Balena
```

Comment:
Maybe we should update?  `Copyright 2017-2020 Balena Ltd.`

Reply:
will do

Reply:
done